client/my-sites/stats/stats-subscribers-page/subsriber-highlights.jsx

```diff
@@ -0,0 +1,97 @@
+import { HighlightCard, Popover } from '@automattic/components';
```

Comment:
We could also try making this a TypeScript file if you'd be interested in the challenge 🙂

Reply:
Definitely, I'll give this a go!